store/generated_key/memstore/memconfig/config.go

```diff
@@ -118,6 +132,34 @@ func (sc *SafeConfig) SetMaxBlobSizeBytes(maxBlobSizeBytes uint64) {
 	sc.config.MaxBlobSizeBytes = maxBlobSizeBytes
 }
 
+func (sc *SafeConfig) GetGETReturnsInstructedStatusCode() (bool, coretypes.VerificationStatusCode) {
+	sc.mu.RLock()
+	defer sc.mu.RUnlock()
+
+	return sc.config.GetReturnsInstructedStatusCode.IsActivated,
+		sc.config.GetReturnsInstructedStatusCode.GetReturnsStatusCode
+}
+
+func (sc *SafeConfig) SetGETReturnsInstructedStatusCode(mode GetReturnsInstructedStatusCode) error {
```

Comment:
please don't use "mode", that's already super overloaded in proxy. Just use `sc` here maybe if you want to be brief?